### ISSUE TITLE
Tenderness along peroneal tendon(s)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11166]
**Preferred term label:**
Tenderness along peroneal tendon(s)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain elicited with palpation along the course of the peroneal tendons, located posterior to the lateral malleolus and extending distally toward the base of the fifth metatarsal

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Peroneal tendon disorder ; https://www.ncbi.nlm.nih.gov/books/NBK544354/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001407
- label: Peroneal tendon tenderness
- parents: [Foot pain (HP:0025238)](https://hpo.jax.org/app/browse/term/HP:0025238)
- pmid: PMID:31335074
- def: Pain elicited with palpation along the course of the peroneal tendons, located posterior to the lateral malleolus and extending distally toward the base of the fifth metatarsal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11166
- synonyms:
